tools/storyteller/StoryReader.java

```diff
@@ -20,13 +29,53 @@ public StoryReader(FileUtils fileUtils) {
     this.fileUtils = fileUtils;
   }
 
-  public ImmutableList<Story> getStories(String path) {
-    if (fileUtils.fileExists(path)) {
-      StoryList stories = (StoryList) fileUtils.readPrototxtUnchecked(path, StoryList.newBuilder());
-      return ImmutableList.copyOf(stories.getStoryList());
-    } else {
+  ImmutableList<Story> getStoriesWithScreenshots(String path) {
+    String storiesFolderPath = path.substring(0, path.lastIndexOf("/"));
```

Comment:
The `stories.prototxt` file does not save screenshots locally, because the size in text encoding is too large. Instead, each item in the history has a `screenshot_filename` field. During running `share` command screenshots are read by filename and saved to Firebase. It looks like it:
```
story {
  startTimeMs: 1548663364233
  endTimeMs: 1548663378257
  project: "Production"
  item {
    timeMs: 1548663374542
    oneliner: " "
    screenshot_filename: "2019-01-28_10:16:14_EET.jpg"
  }
}
```
We needed to get `storiesFolderPath` because `path` looked like it:
```
/path/to/unshared/stories/stories.prototxt
```
After getting `storiesFolderPath` we could to get screenshot by path: 
```
/path/to/unshared/stories/ + 2019-01-28_10:16:14_EET.jpg
```